cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -2087,6 +2095,61 @@ declare namespace Deno {
       options?: KvListOptions,
     ): KvListIterator<T>;
 
+    /**
+     * Add a value into the database queue to be delivered to the queue
+     * listener via {@linkcode Deno.Kv.queueListen}.
+     *
+     * ```ts
+     * const db = await Deno.openKv();
+     * await db.enqueue("bar");
+     * ```
+     *
+     * The `delay` option can be used to specify the delay (in milliseconds)
```

Comment:
Is this the delivery time calculated on the client or on the database. Have you considered making this a `Date` (explicit timestamp) rather than a delay?

Reply:
The delivery time is calculated in the client as `now(UTC) + delay` and is stored in the database [as integer](https://github.com/denoland/deno/blob/main/ext/kv/sqlite.rs#L70).

I was debating whether to make this a `Date`, but went with a delay offset, similar to `setTimeout` and `setInterval`.

Making this an explicit timestamp would add a complication of having to deal with time zones and daylight saving time (e.g. if the time got adjusted between enqueue and dequeue). delay API seems simpler, but I'm open to `Date` if that makes more sense here.

Reply:
I would prefer having a deadline-style parameter here too:

```ts
await db.enqueue("bar", { deliverAt: Date.now() + 86400 + 1000 })
```

Reply:
To avoid the TZ issues, we can wait with the deadline parameter until `Temporal` is shipped - that will have a proper zoned date time

Reply:
Looks like `Temporal.ZonedDateTime` would be ideal for this use case since it automatically adjusts for DST. @lucacasonato - what is the timeline for it?

One option could be to start with `delay` and `deliverAt` based on Unix epoch/UTC, and later add another `Temporal.ZonedDateTime` flavor.

Reply:
Later in the year. I think just `delay` is fine for now